docs/dems.md

```diff
@@ -1,49 +1,70 @@
 # Digital Elevation Models (DEMs)
```

Comment:
i just copied the DEM section from the rtc_product_guide here.

Reply:
Updated the image path for the figures and added the Copernicus DEM info box